server/controllers/routes/user/personal.js

```diff
@@ -0,0 +1,47 @@
+const { compare } = require('bcryptjs');
+
+const personalSchema = require('../../utils/personalSchema');
+const { getPassword } = require('./../../../database/queries/getPassword');
+const { updatePersonalDataQuery } = require('./../../../database/queries/updatePersonalData');
+
+exports.updatePersonal = (req, res, next) => {
+  const {
+    oldPassword, firstName, lastName, email,
+  } = req.body;
+  const { id } = req.user;
+
+  getPassword(req.user.id)
```

Comment:
validate the incoming data before check the password

Reply:
I think validation the data is after checking the password @knanahassouna1 
We want to make sure he is the right user, then we validate and update his data. Right?

Reply:
@knanahassouna1, I have validated data before